library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -745,14 +737,6 @@ data class Conversations(
                 }
             } catch (error: CancellationException) {
                 break
-            } catch (error: StatusException) {
```

Comment:
Shouldn't we be trying to reconnect when these errors happen. If we throw we leave it up to the app to handle reconnects